tests/multithreading/thread_local_storage_08.cc

```diff
@@ -0,0 +1,79 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2008 - 2018 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE.md at
+// the top level directory of deal.II.
+//
+// ---------------------------------------------------------------------
+
+
+// test if ThreadLocalStorage::operator= (const ThreadLocalStorage<T> &&t) moves
```

Comment:
```suggestion
// test if ThreadLocalStorage::operator= (ThreadLocalStorage<T> &&t) moves
```

Reply:
Thank you for the correction. I changed the comment so it says that `ThreadLocalStorage::operator=(ThreadLocalStorage<T> &&t=` moves the shared pointer to the exemplar and not the exemplar directly.